lib/helpstation/processors/active_record_fetcher.rb

```diff
@@ -0,0 +1,11 @@
+module Helpstation
+  module Processors
+    class ActiveRecordFetcher < ByKeyFetcher
+      def self.build(*)
+        super
+      rescue ActiveRecord::NotFoundError
+        raise NotFoundError
```

Comment:
I dont like that this custom error is defined in another file. However, its hard to do anything about it besides merging the files and I guess since the superclass has it then its not so bad